policy-packs/aws-ts-finops/cloudwatch.ts

```diff
@@ -0,0 +1,35 @@
+import * as aws from "@pulumi/aws";
+import {
+  ReportViolation,
```

Comment:
Super nit pick, but I think most (all?) of our existing examples use 4 spaces rather than 2 spaces for indentation.